docs/winc-prerequisites.md

```diff
@@ -0,0 +1,40 @@
+# Windows Container Workloads Prerequisites
+Below are the prerequisites for Windows Container Workloads. Please also see the cloud provider-specific sections that 
+may be relevant.
+
+## Supported Kubernetes Distribution Version
+* OKD/OCP 4.6+ must be used in order to configure Windows VMs
+* OKD/OCP 4.8+ and WMCO 3.0+ must be used in order to configure Bring-Your-Own-Host(BYOH) Windows instances
+
+## Supported Cloud Providers
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* VMware vSphere
+
+## Supported Windows Server versions
+The table below outlines the supported 
+[Windows Server version](https://docs.microsoft.com/en-us/windows/release-health/release-information) based on the cloud
+platform you are using. *These are the only Windows Server versions supported by Windows Containers.* Any other Windows 
+Server version not listed means it is NOT currently supported by Win-C, and will give you errors. Please only use the 
```

Comment:
`Win-C` have we started using this in other places?

Reply:
I recommend revising this as a note.

"Any unlisted Windows Server versions are not supported by Win-C and will cause errors. To prevent these errors, only use the appropriate version according to the Cloud Provider in use."

Reply:
Lets remove `Win-C` everywhere in this PR including the PR and commit messages.

Reply:
I'll change it to Windows Containers instead of Win-C

Reply:
In most cases even saying `Windows containers` is not needed.